data/borderlines.json

```diff
@@ -1,5 +1,51 @@
 {
   "cities": [
+    {
+      "name": "Nepalgunj",
```

Comment:
The city names should be arranged alphabetically.

Reply:
done